rs/sns/testing/src/pocket_ic.rs

```diff
@@ -0,0 +1,127 @@
+use candid::{CandidType, Encode};
+use canister_test::Wasm;
+use ic_nervous_system_integration_tests::pocket_ic_helpers::{
+    add_wasms_to_sns_wasm, install_canister_with_controllers, install_nns_suite,
+};
+use ic_nns_constants::{
+    CYCLES_MINTING_CANISTER_ID, GOVERNANCE_CANISTER_ID, IDENTITY_CANISTER_ID, LEDGER_CANISTER_ID,
+    LEDGER_INDEX_CANISTER_ID, NNS_UI_CANISTER_ID, ROOT_CANISTER_ID, SNS_AGGREGATOR_CANISTER_ID,
+    SNS_WASM_CANISTER_ID,
+};
+use pocket_ic::nonblocking::PocketIc;
+
+pub async fn bootstrap_nns(pocket_ic: &PocketIc) {
+    install_core_nns_canisters(pocket_ic).await;
+    install_frontend_nns_canisters(pocket_ic).await;
+    add_wasms_to_sns_wasm(pocket_ic, false).await.unwrap();
+}
+
+pub async fn install_core_nns_canisters(pocket_ic: &PocketIc) {
+    // TODO: use meaningful initial_balances and neuron_fund_hotkeys
```

Comment:
Not sure if meant to be resolved in this changes itself.

Reply:
For now, default values are sufficient so this is rather a reminder for myself that these might need to be updated in future